src/commands/copyFunctionUrl.ts

```diff
@@ -10,7 +10,7 @@ import { FunctionTreeItem } from '../tree/FunctionTreeItem';
 
 export async function copyFunctionUrl(node?: FunctionTreeItem): Promise<void> {
     if (!node) {
-        node = <FunctionTreeItem>await ext.tree.showTreeItemPicker([FunctionTreeItem.contextValue, FunctionTreeItem.readOnlyContextValue]);
+        node = <FunctionTreeItem>await ext.tree.showTreeItemPicker(/^azFuncFunctionHttp(ReadOnly|)$/i);
```

Comment:
Does copyFunctionUrl only work for ReadOnly functions?

Reply:
It works for non-readonly as well. `(ReadOnly|)` should handle that since there's an empty string after `|`

Reply:
Is this for the Disabled case?

Reply:
Is what for the disabled case?

Reply:
Why have the RegExp to indicate that it is Read-Only or not?  Is it because you don't want copyFunctionUrl to appear in the context menu?  The only other label I can think of is (Disabled)

Reply:
Are you asking why I don't do `/^azFuncFunctionHttp/i`? No strong reason, but I prefer to be explicit. I ran into problems a few times when I didn't extend the RegExp all the way to the end of the string (aka when I didn't have `$` at the end) so this felt safer. Specifically the generic [functions node](https://github.com/Microsoft/vscode-azurefunctions/blob/master/src/tree/FunctionsTreeItem.ts#L16) has a context value of `azFuncFunctions` and I pretty much never meant to match that